web/src/components/organisms/Project/Content/ContentList/index.tsx

```diff
@@ -25,7 +29,11 @@ const ContentList: React.FC = () => {
     <PaddedContent>
       <StyledModelsMenu title={t("Content")} selectModel={selectModel} />
       <ContentChild>
-        <ContentTable />
+        <ContentTableWrapper
```

Comment:
Can you move this Wrapper's parent `ContentChild`'s styles in to the ContentTableWrapper? Currently this organism has styles (which it shouldn't as we discussed) and it makes more sense that the ContentTableWRAPPER has the wrapper that is wrapping its content :P

Reply:
https://github.com/reearth/reearth-cms/blob/25bf6fb1124a2ffa260fcd8fb8de42741f96f325/web/src/components/organisms/Project/Content/ContentList/index.tsx#L26-L32